lib/validators/html-validator.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const fs = require('fs-extra');
+const Validator = require('./validator');
+const util = require('util');
+
+const ERROR_TEMPLATE =
+  `Config file validation error for route '%s' - '%s':
+        * File(s) for HTML rendering not found: %s.`;
+
+/**
+ * Validates the HTML sections of the config file.
+ *
+ * @type {module.HtmlInfoValidator}
+ */
+module.exports = class HtmlInfoValidator extends Validator {
+  constructor() {
+    super();
+  }
+
+  /**
+   * Validates HTML info.
+   *
+   * @param validatorInput input to be validated.
```

Comment:
Nit: If it's written as `@param validatorInput - input to be validated.` it's clearer what the parameter is. Mind the `-` that I added.